std/numeric.d

```diff
@@ -2572,15 +2572,16 @@ GapWeightedSimilarityIncremental!(R, F) gapWeightedSimilarityIncremental(R, F)
 
 /**
 Computes the greatest common divisor of $(D a) and $(D b) by using
-Euclid's algorithm.
+an efficient algorithm such as $(HTTP https://en.wikipedia.org/wiki/Euclidean_algorithm, Euclid's)
+or $(HTTP https://en.wikipedia.org/wiki/Binary_GCD_algorithm, Stein's) algorithm.
```

Comment:
That's exactly why we want to remove the `HTTP` and `HTTPS` macros, they cost a lot more than they actually do save!